content/events/18-19/12urenloop.md

```diff
@@ -0,0 +1,25 @@
+---
+title: zeus@12urenloop
+description: LLopen en rondjes tellen op de 12urenloop
+created_at: 11-04-2019
+time: 29-04-2019 09:00
+end: 29-42-2019 21:00
+facebook: https://www.facebook.com/events/628920360914130/
+location: Sint-Pietersplein
+locationlink: Sint-Pietersplein, Ghent, Belgium
+image: //zeus.ugent.be/zeuswpi/6eVnYRmi.jpg
+---
+
+## Lopen
+
+Een traditie die gestart is een aantal jaar geleden met een moedige 10tal lopers: niet laatste eindigen! Ondertussen hebben we met de gezellige bende van het Werkgroepen en Verenigingen Konvent (onze mama-konvent) een mooie bende samengesprokkeld: samen met CenEka, VVN, Moeder Theepot en natuurlijk ons hoogsteigenlijke [Plantje](https://www.facebook.com/plantje12UL/) maken we er een toffe dag van.
+
+Meer info op <https://www.facebook.com/events/628920360914130/>.
+
+Iedereen welkom, lopen optioneel!
+
+## Helpen
+
+Een traditie die al een pak langer duurt, ook dit jaar verzorgt Zeus de IT van de 12urenloop, met als centerpiece het automatisch telsysteem. Alle extra handen zijn welkom. Zowel in de voorbereiding, als de dag zelf. Zeker de ochtend van het evenement zelf kunnen we nog wat gemotiveerde mensen gebruiken die een paar uur op de knoppen van het manueel telsysteem duwen.
+
+Kom zeker eens hallo zeggen in het [Mattermost-kanaal](https://mattermost.zeus.gent/zeus/channels/12urenloop)!
```

Comment:
Misschien een doodle/form/... maken voor wie wanneer beschikbaar is voor moesten we manual count nodig hebben dat we weten we we kunnen bereiken...

Reply:
@RobbeDP een shiftenlijst

Reply:
ik zou wel niet direct hier in zetten

Reply:
@RobbeDP is iemand anders :stuck_out_tongue: Maar je hebt wel gelijk